lib/ClangImporter/ImportDecl.cpp

```diff
@@ -3918,6 +3918,11 @@ namespace {
                               /*IsCaptureList*/false,
                               Impl.importSourceLoc(decl->getLocation()),
                               name, dc);
+      if (decl->getType().isConstQualified()) {
+        // Note that in C++ there are ways to change the values of const
+        // members, so we don't use WriteImplKind::Immutable storage.
+        result->overwriteSetterAccess(AccessLevel::Private);
```

Comment:
Can we use `setSetterAccess` instead?

Reply:
I don't think so.

https://github.com/apple/swift/blob/59368a6e27a86e198af8617035a7722c7d3b0003/lib/ClangImporter/ImporterImpl.h#L1308
https://github.com/apple/swift/blob/59368a6e27a86e198af8617035a7722c7d3b0003/include/swift/AST/Decl.h#L4823